sklearn/preprocessing/_discretization.py

```diff
@@ -263,15 +263,8 @@ def transform(self, X):
 
         bin_edges = self.bin_edges_
         for jj in range(Xt.shape[1]):
-            # Values which are close to a bin edge are susceptible to numeric
-            # instability. Add eps to X so these values are binned correctly
-            # with respect to their decimal truncation. See documentation of
-            # numpy.isclose for an explanation of ``rtol`` and ``atol``.
```

Comment:
so you decide to ignore this comment? I'll vote +1 because I can't understand this comment.

Reply:
I think the comment is caused by a misunderstanding of how digitize works. I'm using open intervals on the outer boundaries which clearly is stable. I don't see how this code could provide stability for the inner interval boundaries.